docs/user-guide/data.md

```diff
@@ -50,13 +50,69 @@ Special syntax:
 
 ![Advanced search link is findable in the search menu at the right of the full text search field.](../assets/images/search-advanced-link.png)
 
-The advanced search allows you to filter by project, by source type, or by the metadata of source types. Each filter can be standalone or combined. The metadata field can be precisely filtered with criteria such as "contains", "is like", "equals to", "exists" or in case of a date value with "before" or "after".
+The advanced search allows for the creation of complex queries using a graphical widget.
+The widget's contents are then turned into a string representing a Gravsearch (SPARQL) query to be sent to DSP-API.
 
-In addition, for a metadata field that is connected to another source type, it's possible to filter by this second source type.
+A query consists of the following elements:
+- ontology (data model) selection
+- selection of a resource class belonging to the selected ontology (optional)
```

Comment:
We are using the term "data model" in DSP-APP and in the documentation. It would be more consistent if we listed it as follows:

- data model (ontology) selection
- selection of a resource class belonging to the selected data model (optional)

Reply:
fixed in https://github.com/dasch-swiss/dsp-app/pull/470/commits/6598f08778deb0efc93111617e6511af65b18f59